client/pages/Home/Books.js

```diff
@@ -0,0 +1,37 @@
+import React from 'react';
```

Comment:
This should just be the `Home` component. No need for an additional `Books` component